kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README

```diff
@@ -0,0 +1,69 @@
+# Domain in PV sample using WebLogic Deploy Tooling
+
+ You can use [WebLogic Deploy Tooling](https://github.com/oracle/weblogic-deploy-tooling) (WDT) to introspect an already
+ existing domain, such as an on-premises domain, and then create the same domain in a Persistent Volume (PV)
+in Kubernetes. The [Discover Domain Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/discover.md) generates a WDT model YAML file (a metadata representation of the
+domain configuration) and a properties file, which contains a key=value pair for each property in the model.
+This sample updates the domain using both the WDT [Update Domain Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/update.md) and
```

Comment:
@rosemarymarano  we need to include both the create and update, how about:
This sample creates the domain using the WDT [Create Domain Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/create.md) and updates the domain using the WDT [Update Domain Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/update.md).  In both cases the sample uses the [Extract Domain Resource Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/kubernetes.md) to generate the domain.yaml.

Reply:
ok, minor edits, "This sample creates the domain using the WDT [Create Domain Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/create.md) and updates the domain using the [Update Domain Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/update.md).  In both cases, the sample uses the [Extract Domain Resource Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/kubernetes.md) to generate the `domain.yaml` file.